benchmark.js

```diff
@@ -12,6 +12,8 @@ var Benchmark = (function() {
     // to take effect, unless the distribution is too weird
     numRounds: 30,
     roundDelay: 5000, // ms to delay starting next round of tests
+    warmBench: false,
```

Comment:
Yes, because it takes longer to run (by default 10 seconds for each run) and because its results might be different than cold startup.